src/components/Auth/LogoutButton.tsx

```diff
@@ -9,7 +9,7 @@ import * as Auth0 from "../../constants/Auth0";
 import { logout } from "../../redux/reducers/authReducer";
 import { setLoading } from "../../redux/reducers/loadingReducer/index";
 import Button from "../Button";
-import { useAuth0, Auth0Provider } from "react-native-auth0";
+import {getAuth, signOut} from "firebase/auth"
```

Comment:
Remove lines 1, 7, 10, 14-15, and 45-63 (related to old OAuth code).

Reply:
Removed Auth0 code and unnecessary imports.